plugin/documents.py

```diff
@@ -597,13 +598,21 @@ def _unresolved_code_lenses(
 
     # --- textDocument/documentHighlight -------------------------------------------------------------------------------
 
+    def _highlights_key(self, kind: int, multiline: bool) -> str:
+        return "lsp_highlight_{}{}".format(_kind2name[kind], "m" if multiline else "s")
+
     def _clear_highlight_regions(self) -> None:
         for kind in range(1, 4):
-            self.view.erase_regions("lsp_highlight_{}".format(_kind2name[kind]))
+            self.view.erase_regions(self._highlights_key(kind, False))
+            self.view.erase_regions(self._highlights_key(kind, True))
 
     def _is_in_higlighted_region(self, point: int) -> bool:
         for kind in range(1, 4):
-            regions = self.view.get_regions("lsp_highlight_{}".format(_kind2name[kind]))
+            regions = self.view.get_regions(self._highlights_key(kind, False))
+            for r in regions:
+                if r.contains(point):
+                    return True
```

Comment:
Sorry, not an issue but could be shorter?
```suggestion
            if any((r.contains(point) for r in regions)):
                return True
```
especially since it's repeated twice